app/controllers/auth/omniauth_callbacks_controller.rb

```diff
@@ -70,7 +70,9 @@ def check_membership
     when "gitlab"
       if conf["group"].present?
         # Get user's groups.
-        is_member = member_of("https://gitlab.com/api/v4/groups") do |g|
+        server = conf.fetch("server", "")
+        server = server.presence || "https://gitlab.com"
```

Comment:
I think this is simpler, no?

```ruby
server = conf.fetch("server", "https://gitlab.com")
```

Reply:
Nope as "" can be in conf, and we want to avoid empty string and default to "https://gitlab.com".